### ISSUE TITLE
[Question] Uploading my own zip file

### ISSUE THREAD

**Issue by arielwb** [lambci/git-lambda-layer#31]
Hello!

I'm trying to use your project in my lambda, but due to restriction of my company's infrastructure, I need to deploy the git layer to our own s3.

I uploaded [this zip file](https://github.com/lambci/git-lambda-layer/blob/master/lambda1/layer.zip) to our s3 but it did not work, git is still unavailable.

My question is: Is this zip file supposed to work used like this?
It totally can be something on my end, but it appears that everything is as it should, so I came here to exclude this possibility :)

**Comment by mhart** [comment]
Yep, that should work (make sure you're using the appropriate zipfile for your Lambda runtime) – you can just copy the commands here to publish your own layer: https://github.com/lambci/git-lambda-layer/blob/master/lambda2/publish.sh

**Comment by arielwb** [comment]
Thank you for your reply!

I did manage to go a step further than I was before, but I'm not there yet hahahhah

I'm using your [build script](https://github.com/lambci/git-lambda-layer/blob/master/lambda2/build.sh) to generate the zip file, and I'm getting the following error when I try to clone a repo:

```
/opt/libexec/git-core/git-remote-https: error while loading shared libraries: libcurl.so.4: cannot open shared object file: No such file or directory
```

I'm thinking that maybe I'm not using the correct docker image for my runtime?
My Lambda runs on node 12.x, and from what I got from the repo, the image you use to build is `lambci/yumda:2` right?

Thank you so much! Any help will be appreciated :)

**Comment by Mickaz89** [comment]
Same error for me , i uploaded the zip but git still unavailable

**Comment by Mickaz89** [comment]
?

**Comment by leumifintech** [comment]
?

**Comment by leumifintech** [comment]
?

**Comment by jlpl999** [comment]
Any help on this, upload layer.zip to S3, go to Lambda/Layer/Add Layer and point to S3 location.
Lambda python is still error out with error, 
My env is restricted, so there is not possible to use the share ARN that listed. 
Please help, really appreciated
{
  "errorMessage": "Unable to import module 'lambda_function': No module named 'git'",
  "errorType": "Runtime.ImportModuleError",
  "stackTrace": []
}
